pkg/cmd/server/origin/run_components.go

```diff
@@ -378,6 +378,14 @@ func (c *MasterConfig) RunSecurityAllocationController() {
 	repair := securitycontroller.NewRepair(time.Minute, kclient.Namespaces(), uidRange, etcdAlloc)
 	if err := repair.RunOnce(); err != nil {
 		// TODO: v scary, may need to use direct etcd calls?
+		// If the security controller fails during RunOnce it could mean a
+		// couple of things:
+		// 1. an unexpected etcd error occurred getting an allocator or the namespaces
+		// 2. the allocation blocks were full - would result in an admission controller that is unable
+		//	  to create the strategies correctly which would likely mean that the cluster
+		//	  would not admit pods the the majority of users.
```

Comment:
As far as admission goes, it is always fail safe.  When creating providers it will collect the errors during the creation process and then attempt to validate against any good providers as well as log any creation errors.  In the case of no available providers it will provide a Forbidden response just as if it failed to validate.

You could, in fact, bring up the master.  With a default configuration only administrators would be able to create pods since the privileged SCC doesn't rely on allocations.  Seems like it would just be a bad experience though and that we should go ahead and fail for the administrator so they aren't stuck guessing why no one can create pods.

cc @smarterclayton